seed-data.json

```diff
@@ -844,7 +846,8 @@
       "state": "CLOSED",
       "project_name": "Stellar Study Group [Core]",
       "title": "Start here",
-      "body": "",
+      "language": "English",
```

Comment:
language here means programming language but my bad not giving you context. In fact I think we will not use this for a while